scripts/test-extensions-unit.bat

```diff
@@ -21,42 +21,42 @@ if "%ADS_TEST_GREP%" == "" (
 echo ***************************************************
 echo *** starting admin tool extension windows tests ***
 echo ***************************************************
-call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\admin-tool-ext-win --extensionTestsPath=%~dp0\..\extensions\admin-tool-ext-win\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --disableExtensions --remote-debugging-port=9222
+call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\admin-tool-ext-win --extensionTestsPath=%~dp0\..\extensions\admin-tool-ext-win\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --disable-telemetry --disable-crash-reporter --disable-updates --skip-getting-started --disable-inspect --disable-extensions --remote-debugging-port=9222
```

Comment:
So --disable-extensions is fine to use even when testing the extensions themselves? I'm assuming that in this case it still loads the extension passed in on the extensionDevelopmentPath and extensionTestsPath?

Reply:
Yep, it stops any other extension from loading